lib/index.js

```diff
@@ -2258,6 +2258,28 @@ exposeMethod('uniqBy');
  */
 
 Stream.prototype.uniq = function () {
+    if (!_.isUndefined(typeof Set)) {
```

Comment:
We can use `!_.isUndefined(_global.Set)`. We detect `Symbol` for iterator support in the same way.